docs/src/whatsnew/latest.rst

```diff
@@ -44,6 +47,10 @@ This document explains the changes made to Iris for this release
    Winter - December to February) will be assigned to the preceding year (e.g.
    the year of December) instead of the following year (the default behaviour).
    (:pull:`5573`)
+   
+   #. `@HGWright`_ added :func:`~iris.coords.Coord.ignore_axis` to allow manual
```

Comment:
```suggestion
   #. `@HGWright`_ added :attr:`~iris.coords.Coord.ignore_axis` to allow manual
```

Reply:
For further details, see [here](https://www.sphinx-doc.org/en/master/usage/domains/python.html#role-py-attr).